src/ast/query.rs

```diff
@@ -899,6 +899,14 @@ impl fmt::Display for ExprWithAlias {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub struct TableFunctionArgs {
+    pub args: Vec<FunctionArg>,
+    pub settings: Option<Vec<Setting>>,
```

Comment:
Can we include a description of the syntax and link to the docs? that would help folks in navigating the code later on